inc/components/class-image.php

```diff
@@ -124,11 +124,22 @@ public static function register_crop_sizes( array $crop_sizes ) {
 	public function set_post_id( $post_id ) {
 		// Get the URL.
 		$attachment_id  = get_post_thumbnail_id( $post_id );
```

Comment:
There are two spaces after `$attachment_id` and PHPCS is flagging it as a warning.